src/orange/orangeinp/detail/CsgUnitBuilder.cc

```diff
@@ -49,6 +50,16 @@ BoundingZone const& CsgUnitBuilder::bounds(NodeId nid) const
     return iter->second.bounds;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Insert transform with simplification and  deduplication.
```

Comment:
Extra space between "and" and "deduplication" --- not sure if Doxygen can strip that out